### ISSUE TITLE
[icetray] Documentation for I3Frame pybindings (Trac #1584)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1576]
Tried to point someone to docs on I3Frame python bindings and couldn't easily find it.  Maybe they're buried somewhere in here : http://software.icecube.wisc.edu/offline_trunk/projects/icetray/index.html ...but those docs need a re-org.  There's a lot...but they're kinda terrible, especially considering this is our framework for most of IceCube's software.

Migrated from https://code.icecube.wisc.edu/ticket/1584
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:26", 
    "description": "Tried to point someone to docs on I3Frame python bindings and couldn't easily find it.  Maybe they're buried somewhere in here : http://software.icecube.wisc.edu/offline_trunk/projects/icetray/index.html ...but those docs need a re-org.  There's a lot...but they're kinda terrible, especially considering this is our framework for most of IceCube's software.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067086520250", 
    "component": "combo core", 
    "summary": "[icetray] Documentation for I3Frame pybindings", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-03-10T16:23:32", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160310T16:30:41: david.schultz commented:

> Part of the problem might be duplicates and partial information in several places.  I don't really get why there's a "general overview" that's fairly detailed in some areas, then a "detailed documentation" that repeats stuff.
> 
> I guess someone needs to sit down and write a new outline, and then get a bunch of people to fit the existing docs to that outline (removing duplicates where possible).

**Comment by icecube-trac** [comment]
Trac update at 20160310T16:34:43: olivas commented:

> Yeah.  I made a pass several months (to maybe a year ago) and took care of some of the duplicates and made a bunch of cleanups. Just needs another good pass and likely a *complete* reorganization of the structure.  It's nearly impossible to find any information, in my opinion, unless you already know where it's at.

**Comment by icecube-trac** [comment]
Trac update at 20160328T20:36:06: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160421T18:43:02: 
* nega changed resolution from "" to "fixed"
* nega changed status from "accepted" to "closed"
* nega commented:

> In changeset 2522:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2522"
> fix doc deployment for releases
> 
> closes #1576
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:15:03: nwhitehorn commented:

> In changeset 170245:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170245"
> fix doc deployment for releases
> 
> closes #1576
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:26: 
* olivas commented:

> Milestone offline V16-03-00 deleted


* olivas changed milestone from "offline V16-03-00" to ""
